src/editor.js

```diff
@@ -0,0 +1,154 @@
+import { registerPlugin } from '@wordpress/plugins';
+import { PluginSidebar, PluginSidebarMoreMenuItem } from '@wordpress/edit-site';
+import { blockDefault } from '@wordpress/icons';
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	Button,
+	__experimentalVStack as VStack,
+	__experimentalText as Text,
+	__experimentalHeading as Heading,
+	PanelBody,
+	TextControl,
+} from '@wordpress/components';
+import { store as noticesStore } from '@wordpress/notices';
+import { useDispatch, useSelect } from '@wordpress/data';
+
+const ExportTheme = () => {
+	const { createErrorNotice } = useDispatch( noticesStore );
+	const [ theme, setTheme ] = useState( {
+		name: '',
+		description: '',
+		uri: '',
+		author: '',
+		author_uri: '',
+	} );
+
+	useSelect( ( select ) => {
+		const themeData = select( 'core' ).getCurrentTheme();
+		setTheme( {
+			name: themeData.name.raw,
+			description: themeData.description.raw,
+			author: themeData.author.raw,
+			author_uri: themeData.author_uri.raw,
+			theme_uri: themeData.theme_uri.raw,
+		} );
+	}, [] );
+
+	const handleSubmit = () => {
+		fetch( '/wp-json/create-block-theme/v1/export', {
```

Comment:
We would need to make this URL absolute. Otherwise it will fail when WordPress is not served from the root of the domain.

Reply:
`apiFetch` takes care of this, so it should be fixed with my latest commit.

Reply:
Thank you!